Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp

```diff
@@ -149,12 +149,17 @@ static ALWAYS_INLINE CCallHelpers::Address callFrameAddr(CCallHelpers& jit, intp
         return CCallHelpers::Address(GPRInfo::callFrameRegister, offsetFromFP);
     }
 
+    if (isARM()) {
+        // For now, we can just solve this in the macro assembler ...
+        return CCallHelpers::Address(GPRInfo::callFrameRegister, offsetFromFP);
+    }
+
     auto addr = Arg::addr(Air::Tmp(GPRInfo::callFrameRegister), offsetFromFP);
-    if (addr.isValidForm(Width64))
+    if (addr.isValidForm(pointerWidth()))
         return CCallHelpers::Address(GPRInfo::callFrameRegister, offsetFromFP);
     GPRReg reg = extendedOffsetAddrRegister();
     jit.move(CCallHelpers::TrustedImmPtr(offsetFromFP), reg);
-    jit.add64(GPRInfo::callFrameRegister, reg);
+    jit.addPtr(GPRInfo::callFrameRegister, reg);
```

Comment:
Ditto. Ptr width is not register width.

Reply:
This is fixed elsewhere, but this register happens to really hold a pointer, so I kept this the same.